src/components/dateTimePicker/DateTimePicker.driver.ts

```diff
@@ -0,0 +1,31 @@
+import {fireEvent} from '@testing-library/react-native';
+import {
+  ComponentProps,
+  ExpandableOverlayDriver,
+  ButtonDriver
+} from '../../testkit';
+
+export const DateTimePickerDriver = (props: ComponentProps) => {
+  const {renderTree, testID} = props;
+  const expandableOverlayDriver = ExpandableOverlayDriver({...props, testID: `${testID}.overlay`});
+  const openPicker = () => {
+    expandableOverlayDriver.open();
+  };
+  const discardPicker = () => {
+    ButtonDriver({...props, testID: `${testID}.cancel`}).press();
+  };
+  const submitSelection = () => {
```

Comment:
Although `submitSelection` is not bad, don't we usually go with `done` or maybe `select`?

Reply:
same here. I'll change it to select.